contao/classes/Frontend.php

```diff
@@ -442,13 +442,9 @@ public static function addToUrl($strRequest, $blnIgnoreParams=false, $arrUnset=a
 		if (\Config::get('addLanguageToUrl'))
```

Comment:
I know they are mapped but I guess we should still use the new way in the legacy code as well if we are adjusting things. Of course we don't adjust a usage in e.g. the news bundle but we have adjusted the whole `getRootPageFromUrl()` method with new code so it feels strange to use a "legacy" setting :D